### PR TITLE
Consolidate text between UG and args spec (part 1 of 3)

### PR DESCRIPTION
**Pull request by emlys**
This includes:
- Carbon
- Coastal Blue Carbon 
- Coastal Blue Carbon Preprocessor
- Coastal Vulnerability
- Crop Production Percentile
- Crop Production Regression
- DelineateIt

### CODE REVIEW

File: source/conf.py [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
 # -- General configuration -----------------------------------------------------
+print('in conf.py')
+if not os.path.exists('../invest-sample-data'):
+    print('make get sampledata')
+    subprocess.run(['make', '-C', '..', 'get_sampledata'])
+    print('done')
+if not os.path.exists('invest-sample-data/pollination/landcover_biophysical_table_modified.csv'):
+    print('make prep sampledata')
+    subprocess.run(['make', '-C', '..', 'prep_sampledata'])
+    print('done')
```
[the remark this answers is not in the document]
Comment (emlys) [reply source/conf.py:18]: removed these

File: source/input_types.rst [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+InVEST models accept many different types of spatial and non-spatial data. We categorize them into 12 types:
+
+.. _number:
+
+number
+------
+A scalar value. You may enter numbers in integer or floating-point format. Fractions are not allowed.
+
+Units
+~~~~~
```
[the remark this answers is not in the document]
Comment (emlys) [reply source/input_types.rst:14]: see answer in #34

File: source/input_types.rst [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+.. _raster:
+
+raster
+------
+A spatial data file that stores data in pixels, which each have a numeric value.
+InVEST accepts all GDAL-supported raster formats, including GeoTiff (.tif) and many more.
+For the complete list, see https://gdal.org/drivers/raster/index.html.
+
+Bands
+~~~~~
```
[the remark this answers is not in the document]
Comment (emlys) [reply source/input_types.rst:80]: see #34

File: source/coastal_vulnerability.rst [hunk 4]
```diff
@@ -256,121 +256,61 @@ Consequently, model outputs cannot be used to quantify the exposure to erosion a
...
+     The AOI instructs the model to plot shore points on all **landmass** coastline within this AOI polygon. When drawing the AOI polygon, make sure to exclude any part of the landmass that should not be analyzed.
 
-     Name: File can be named anything, but no spaces in the name
-     File type: polygon vector (e.g. .shp, .gpkg, .geojson)
-     Sample path (default): \InVEST\CoastalVulnerability\landmass_polygon.shp
+     When preparing other input data, it is *not* recommended to clip GIS datasets to the exact boundary of the AOI. Many of the model functions require searching for the presence of layers at certain distances around the coastline, and that requires having data coverage extend beyond the AOI _(is there a minimum distance that the data should extend beyond the AOI?)_. The model will appropriately handle all clipping and projecting of larger datasets as needed. The model uses the AOI's projection to transform the projection of other input data as needed.
```
[the remark this answers is not in the document]
Comment (emlys) [reply source/coastal_vulnerability.rst:266]: Ah okay, thanks. Yea this was a note to self that slipped through. I'll just take it out, we can expand on the details later.

**Review by davemfish — APPROVED** [review]
These chapters look great! Very exciting!

File: source/carbonstorage.rst [hunk 5]
```diff
@@ -42,6 +42,12 @@ Outputs of the model are expressed as Mg of carbon per pixel, and if desired, th
...
 
+The value of carbon sequestration over time for a given parcel *x* is:
+
+.. math:: value\_seq_x=V\frac{sequest_x}{yr\_fut-yr\_cur}\sum^{yr\_fut-yr\_cur-1}_{t=0}\frac{1}{\left(1+\frac{r}{100}\right)^t\left(1+\frac{c}{100}\right)^t}
+   :label: carbon_value
+
```
Comment (dcdenu4) [inline source/carbonstorage.rst:49]: Should the variables in this equation be defined afterwards? Like: where `V` is this and that ,`c` is that and this, etc...?

File: source/carbonstorage.rst [hunk 6]
```diff
@@ -71,17 +77,25 @@ Finally, while most sequestration follows a nonlinear path such that carbon is s
...
-   * **lucode**: Unique integer for each LULC class (e.g., 1 for forest, 3 for grassland, etc.) *Every value in the LULC map MUST have a corresponding "lucode" value in the Carbon Pool table.*
-   * **c_above**: Carbon density in aboveground biomass [units: megagrams/hectare]
-   * **c_below**: Carbon density in belowground biomass [units: megagrams/hectare]
-   * **c_soil**: Carbon density in soil [units: megagrams/hectare]
-   * **c_dead**: Carbon density in dead matter [units: megagrams/hectare]
+- :investspec:`carbon lulc_fut_year`
+
+- :investspec:`carbon do_redd`
+
+- :investspec:`carbon lulc_redd_path`
```
Comment (dcdenu4) [inline source/carbonstorage.rst:96]: so clean!

File: source/crop_production.rst [hunk 7]
```diff
@@ -25,19 +25,19 @@ Expanding agricultural production and closing yield gaps is a key strategy for m
 The Model
 =========
 
-The InVEST crop production model is divided into a percentile based yield model, covering 175 crops worldwide, and a regression based model that accounts for fertilization rates on 12 crops.  These models are deployed to the end user as the "percentile" and "regression" models.
+The InVEST crop production model is divided into a percentile based yield model, covering 175 crops worldwide, and a regression based model that accounts for fertilization rates on 10 crops. These models are deployed to the end user as the "percentile" and "regression" models.
```
Comment (dcdenu4) [inline source/crop_production.rst:28]: Farewell to the two crops that never were, we hardly knew yee

File: source/delineateit.rst [hunk 8]
```diff
@@ -16,26 +16,68 @@ Introduction
...
+
+Detect Pour Points
+^^^^^^^^^^^^^^^^^^
+A pour point is a point where water flows off the defined area of the flow direction map, either off the edge of the raster or into a nodata pixel.
+If the Detect Pour Points option is selected, the model will place a pour point in the center of each
+
+
+Snap Points to Nearest Stream
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If an outlet feature does not lie directly on a stream, a very small watershed will be generated which is usually not correct. DelineateIt can snap point outlet features to the nearest stream to make more robust watersheds. To do this, the tool constructs a stream map,  and then relocates watershed outlet points to the nearest stream.
```
Comment (dcdenu4) [inline source/delineateit.rst:40]:
```suggestion
If an outlet feature does not lie directly on a stream, a very small watershed will be generated which is usually not correct. DelineateIt can snap point outlet features to the nearest stream to make more robust watersheds. To do this, the tool constructs a stream map, and then relocates watershed outlet points to the nearest stream.
```

**Review by dcdenu4 — APPROVED** [review]
Hey @emlys , I just had some trivial suggestions. I'll approve ahead of time because they're so trivial. Looks great!

File: source/carbonstorage.rst (hunk 5 above)
Comment (emlys) [reply source/carbonstorage.rst]: Sure, I added variable definitions. Though, I didn't change this equation, just moved it from a different section of the text. There are many other places in the UG where variables aren't defined, or are re-used to mean different things, and fixing all that will be a bigger project.

File: source/coastal_blue_carbon.rst [hunk 9]
```diff
@@ -665,122 +633,51 @@ sequestration if economic data is provided.
...
+
+- :investspec:`coastal_blue_carbon.coastal_blue_carbon landcover_transitions_table`
+  The Coastal Blue Carbon preprocessor exists to help create this table for you. You must edit the ``transitions_[suffix].csv`` preprocessor output as described in *Step 1 Preprocessing Outputs* before it can be used by the main model.
+
+  Columns:
+
+  - :investspec:`coastal_blue_carbon.coastal_blue_carbon landcover_transitions_table.columns.lulc-class`
+  - :investspec:`coastal_blue_carbon.coastal_blue_carbon landcover_transitions_table.columns.[LULC CODE]`
+
+- :investspec:`coastal_blue_carbon.coastal_blue_carbon landcover_snapshot_csv` The raster with the earliest chronological year will be used as the baseline raster. If rasters provided in this table have different extents or resolutions, they will be resampled to the minimum resolution of the set of rasters, and clipped to the intersection of all of the bounding boxes. If you are only interested in the standing stock of carbon at a single year, then only provide a single row in this table. All rasters provided in this table must be in a projected coordinate system with units in meters. ??
```
[the remark this answers is not in the document]
Comment (emlys) [reply source/coastal_blue_carbon.rst]: Ah, I think I was wondering if the units really needed to be in meters. Doesn't need to be answered now. I removed the ` ??`

**Review by phargogh — APPROVED** [review]
OK great!

**Pull request merged**